repository_service_tuf_api/api/metadata.py

```diff
@@ -31,6 +31,26 @@ def post(payload: metadata.MetadataPostPayload):
     return metadata.post_metadata(payload)
 
 
+@router.put(
+    "/",
+    summary="Force a new version of online metadata role(s).",
+    description=(
+        "Force a new version of online metadata role(s). If the roles list is "
+        "empty all roles will be updated. The new metadata version(s) will "
+        "have extended expiration which will equal to: "
```

Comment:
Same about "extended" here.

Reply:
Check my comment to your questions above.